src/main/java/org/sagebionetworks/web/client/widget/entity/EntityPageTop.java

```diff
@@ -329,7 +329,7 @@ public void handleWikiReload(String wikiPageId) {
 			setArea(EntityArea.WIKI, wikiPageId);
 			view.configureProjectActionMenu(bundle, wikiPageId);
 		} else {
-			DisplayUtils.showErrorMessage("Failed to handle Wiki reload.");
+			view.configureFileActionMenu(bundle, wikiPageId);
```

Comment:
I suppose since a Folder also uses the fileActionMenuContainer, this will fix for both files and folders.